src/main/java/no/nav/veilarboppgave/config/ClientConfig.java

```diff
@@ -72,16 +72,11 @@ public OppgaveClient oppgaveClient(ContextAwareService contextAwareService) {
     }
 
     @Bean
-    public VeilarbpersonClient veilarbpersonClient(ContextAwareService contextAwareService) {
-        String safCluster = isProduction() ? "prod-fss"  : "dev-fss";
-        Supplier<String> userTokenSupplier = contextAwareService.contextAwareUserTokenSupplier(
-                downstreamVeilarbperson(safCluster)
+    public VeilarbpersonClient veilarbpersonClient(EnvironmentProperties properties, AzureAdMachineToMachineTokenClient tokenClient) {
+        return new VeilarbpersonClientImpl(
+                properties.getVeilarbpersonUrl(),
+                () -> tokenClient.createMachineToMachineToken(properties.getVeilarbpersonScope())
```

Comment:
Er det mulig å bruke obo-token med brukerkontekst her istedetfor matchine-to-machine token?

Reply:
Gjorde tilsvarende endringer nå :)